### PR TITLE
Chapter 3: Mostly Static_pages

### PR DESCRIPTION
**Pull request by nvquoctuan**
Ready
Screenshot from 2019-10-23 08-58-10

### CODE REVIEW

**Comment by nvquoctuan** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -10,6 +10,8 @@
   </head>
 
   <body>
+    <%= link_to "English", locale: "en"%>
```
Comment (phongnvk-0916) [inline app/views/layouts/application.html.erb]: i18n

File: app/views/static_pages/about.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<% provide(:title, (t ".title")) %>
+<h1><%= t ".title" %></h1>
+<p>
+  The <a href="https://www.railstutorial.org/"><em>Ruby on Rails
+  Tutorial</em></a> is a
```
Comment (phongnvk-0916) [inline app/views/static_pages/about.html.erb]: i18n

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root 'static_pages#home'
+
+  get 'static_pages/help'
```
Comment (phongnvk-0916) [inline config/routes.rb]: ' => "

File: config/locales/vi.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+vi:
+  layouts:
+    application:
+      title: 'Ứng dụng mẫu hướng dẫn Ruby on Rails'
```
Comment (phongnvk-0916) [inline config/locales/vi.yml]: ' => "

File: config/database.yml [hunk 5]
```diff
@@ -13,8 +13,8 @@ default: &default
   adapter: mysql2
   encoding: utf8mb4
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  username: root
-  password:
+  username: demo_user
```
Comment (phongnvk-0916) [inline config/database.yml]: gitignore file này nhé. Lộ hết pass db rồi :D

**Pull request closed without merging**